ask-watch.py

```diff
@@ -3,8 +3,35 @@
 import ask
 import os
 
+
 os.system('clear')
-print('⚡' + '\033[92m' + '️Ask' + '\033[0m' + ' Watcher')
+
+
+def style_string(text,color='', bold=False):
```

Comment:
- Put spaces after commas.

- Rename to: `style_print()`

- Since style_string() this is only used when printing to the console, maybe the function could print the text out instead of returning it?